_posts/2020-02-04-planning-ahead-for-v1.0.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "What would be kadalu v1.0?"
+author: sankarshan
+layout: post
+description: "In the journey from v0.5 to v1.0 we aim to work through the tasks of maturing the set of capabilities which we consider to be required as part of the general availability of the release. We also intend to focus on reaching out to the users and engaging with them to understand how to streamline the switching over from Heketi to a Kadalu based deployment of Gluster as persistent storage."
+---
+
+
+GlusterFS (https://github.com/gluster/glusterfs) which started as distributed file storage in a standalone manner has had overwhelming user acceptance as preferred persistent storage backend in the cloud. The Heketi project (https://github.com/heketi/heketi) has contributed immensely by enabling a simple ReSTful management interface for volume lifecycle management. This opened up a range of cloud services to dynamically provision storage.
```

Comment:
We can change these as links `[GlusterFS](https://github.com/gluster/glusterfs)`

Reply:
Done